routes/web.php

```diff
@@ -14,7 +14,8 @@
 
 Route::group(['middleware' => 'locale'], function() {
     Route::get('lang/{lang}', 'LangController@changeLang')->name('lang');
+    Route::post('add-user/{id}', 'TaskController@addUser')->name('taskAddUser');
+    Route::delete('delete-user/{task_id}/{user_id}', 'TaskController@deleteUser')->name('taskDeleteUser');
```

Comment:
Tên router dạng snake case
```suggestion
    Route::post('add-user/{id}', 'TaskController@addUser')->name('task.add_user');
    Route::delete('delete-user/{task_id}/{user_id}', 'TaskController@deleteUser')->name('task.delete_user');
```